.github/workflows/vast.yml

```diff
@@ -122,6 +111,17 @@ jobs:
           brew --version
           brew install libpcap tcpdump rsync pandoc apache-arrow pkg-config ninja ccache gnu-sed
 
+      - name: Compile flatbuffers
+        if: matrix.os.tag == 'Ubuntu' || matrix.os.tag == 'macOS'
```

Comment:
Can't you just do `brew install --HEAD flatbuffers` on macOS?

Reply:
This is a platform-agnostic way that adds minimal overhead. I went initially down the rabbit hole of trying to get a PPA for Linux, but that didn't exist.

If you think we need to change the GH workflow, let's do this in a separate PR.

Reply:
I think we should include flatbuffers as a submodule if we're not using a stable version, but you're correct that this belongs in a separate PR.

Let's talk about this again when merging the epic branch into master.